en/mulesoft/modules/ROOT/pages/ms_composer_gmail_reference.adoc

```diff
@@ -29,7 +29,7 @@ include::_partials/trigger-definition.adoc[]
 
 New email::
 
-This trigger fires when a new email is received in the user’s inbox.
+This trigger fires on all emails (received, sent, or drafts), if no https://developers.google.com/gmail/api/guides/labels[filter] is specified.
```

Comment:
```suggestion
If you don't specify a  https://developers.google.com/gmail/api/guides/labels[filter^], this trigger fires on all emails (received, sent, or drafts).
```

Reply:
This trigger fires on all emails (received, sent, or drafts) if you don't specify a  https://developers.google.com/gmail/api/guides/labels[filter^].